apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatInfoView.kt

```diff
@@ -748,10 +752,40 @@ private fun MessageButton(openedFromChatView: Boolean, chat: Chat, contact: Cont
   )
 }
 
+@Composable
+fun SearchButton(chat: Chat, contact: Contact, close: () -> Unit) {
+  InfoViewActionButton(
+    icon = painterResource(MR.images.ic_search),
+    title = generalGetString(MR.strings.info_view_search_button),
+    disabled = !contact.ready || chat.chatItems.isEmpty(),
+    onClick = {
+      close.invoke()
+      withBGApi {
+        openDirectChat(chat.remoteHostId, contact.contactId, chatModel)
```

Comment:
why do we need to call `openDirectChat` here? It's already the one opened - `close` should suffice

Reply:
did a quick check - it seems to be unnecessary
```suggestion
```

Reply:
it may have been needed in previous implementation where chat info could be opened from contacts tab and there was no chat loaded into model already, but now we only allow to open chat info from already loaded chat